.github/workflows/ci.yaml

```diff
@@ -467,7 +467,8 @@ jobs:
   deploy-demo-env:
     name: Deploy updated images to the demo environment
     runs-on: self-hosted
-    if: github.event_name == 'release' || (github.event_name == 'push' && github.ref_name == 'main') || github.event_name == 'workflow_dispatch'
+    if: ${{ false }}
```

Comment:
we could use an actions variable, https://docs.github.com/en/actions/learn-github-actions/variables so we don't need to commit next time we want to activate/deactivate

Reply:
Damn! Unfortunately the `env` value is not available in the `if` at `job` level: https://github.com/actions/runner/issues/1189
It can only be used in the `step` level.
Does it make sense to put it in the last step of the involved jobs?
It would work, but many the initial steps would be executed, and I don't want to put it everywhere...

Reply:
Variables are accepted though...